workshops/models.py

```diff
@@ -369,6 +364,14 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+def is_admin(user):
+    if user is None or user.is_anonymous():
+        return False
+    else:
+        is_admin = user.groups.filter(name='administrators').exists()
+        return is_admin or user.is_staff or user.is_superuser
```

Comment:
1. OK, I've changed that line to `return has_admin_group or user.is_superuser`. 
2. :+1: